crawler/file_handler.py

```
@@ -73,20 +74,142 @@ def _extract_pdf(path: Path) -> tuple[str, str]:
+
+    표는 LIST_HEADER(row/col)로 셀을 특정하고 '헤더 | 셀1 | 셀2' 형식으로 변환.
+    pyhwp 미설치 또는 파싱 실패 시 빈 문자열 반환.
+    """
+    try:
+        from hwp5.xmlmodel import Hwp5File
+    except ImportError:
+        return ""
+
+    try:
+        hw = Hwp5File(str(path))
+        sec = list(hw.bodytext.sections)[0]
```

Ruff (0.15.11)

[warning] Prefer `next(iter(hw.bodytext.sections))` over single element slice

Replace with `next(iter(hw.bodytext.sections))`

(RUF015)